remote_client.md

```diff
@@ -0,0 +1,62 @@
+# Podman remote-client tutorial
+
+### What is the remote-client
+
+First and foremost, the remote-client is under heavy development.  We are adding new
+commands and functions frequently.  We also are working on a rootless implementation that
+does not require privileged users.
+
+The purpose of the Podman remote-client is to allow users to interact with a Podman "backend"
+while one a separate client.  The command line interface of the remote client is exactly the
+same as the regular Podman for commands that are implemented.
+
+### What you need
+To use the remote-client, you will need a binary for your client and a Podman "backend".  In
+this context, a Podman backend is a Linux system with Podman installed on it and the varlink
+service activated.  You will also need to be able to ssh into this system as a user with privileges
+to the varlink socket (more on this later).
+
+### Building the remote client
+At this time, the remote-client is not being packaged for any distribution.  It must be built from
+source.  To set up your build environment, see [Installation notes](install.md) and follow the
+section *Building from scratch*.  Once you can successfully build the regular Podman binary, you
+can build the remote-client.
+```
+$ make podman-remote
+```
+Like building the regular Podman, the resulting binary will be in the *bin* directory.
+
+### Running the remote client
+The remote-client requires a little bit of setup before it will work properly.
+
+#### Remote side set up
+Before running the remote-client, you must make sure Podman is installed.  Then make sure to start
+the Podman varlink socket.
```

Comment:
Note that not all distributions support that, e.g., openSUSE. Can we point to a document/post/artifact that describes the role of systemd?